index.html

```diff
@@ -14,46 +14,258 @@
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
+      </section>
+
+      <!-- headline section -->
+      <section class="headline-section">
+        <div class="content">
+          <h1 class="author">
+            I’m Ismail <br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p class="article">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="let-section">
+          <p class="connect">LET’S CONNECT</p>
+          <img src="img/Ellipse 12.png" alt="Eclipse-image" />
+        </div>
+
+        <div class="eclipse-13">
+          <img src="img/Ellipse 13.png" alt="Eclipse-image" />
+        </div>
+
+        <ul class="social-icons">
+          <li><span class="iconify" data-icon="iconoir:github"></span></li>
+          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
+          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
+          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
+          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
+        </ul>
+
+        <div class="eclipse-4">
+          <img src="img/Ellipse 4.png" alt="Eclipse-image" />
+          <img src="img/Rectangle 5.png" alt="Eclipse-image" />
+        </div>
+      </section>
+    </section>
+
+    <!-------------------------------------- middle-section ---------------------------------------->
+
+    <section class="middle-section">
+      <!-- tonic section 1 -->
+      <section class="tonic-section">
+        <div class="tonic-image"></div>
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <h2 class="canopy">CANOPY</h2>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
         </p>
-      </div>
-
-      <p class="connect">LET’S CONNECT</p>
-
-      <div class="social-icons">
-        <span class="iconify" data-icon="iconoir:github"></span>
-        <span class="iconify" data-icon="eva:linkedin-fill"></span>
-        <span class="iconify" data-icon="fontisto:angelist"></span>
-        <span class="iconify" data-icon="ei:sc-twitter"></span>
-        <span class="iconify" data-icon="bx:bxl-medium"></span>
-      </div>
-    </main>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javaScript</button></li>
+        </ul>
+        <button class="btn">See Project</button>
```

Comment:
- [ ] Please use figma's design for the hover and click animations on this buttons.
## Original
![image](https://user-images.githubusercontent.com/28903404/143720895-6a57a179-12f8-4bd1-bc28-390e0ffb9bbf.png)

## Yours
![WrongButton](https://user-images.githubusercontent.com/28903404/143720934-c628b736-83a5-428c-9028-c05ec5eef5d0.gif)